test_util/aws.py

```diff
@@ -336,18 +322,8 @@ def delete(self):
         self.stack = self.boto_wrapper.resource('cloudformation').Stack(self.stack.stack_id)
         self.stack.delete()
 
-    @retrying.retry(wait_fixed=3000, stop_max_delay=600 * 1000,
-                    retry_on_result=lambda res: res is False)
-    @retry_boto_rate_limits
     def get_vpc_host_ips(self):
-        expected_instances = int(self.get_parameter('ClusterSize'))
-        reservations = self.boto_wrapper.client('ec2').describe_instances(Filters=[{
-            'Name': 'tag-value', 'Values': [self.stack.stack_name]}])['Reservations']
-        log.debug('Reservations for {}: {}'.format(self.stack.stack_id, reservations))
-        instances = [instance for res in reservations for instance in res['Instances']]
-        if len(instances) != expected_instances:
-            return False
-        return instances_to_hosts(instances)
+        return instances_to_hosts(self.get_auto_scaling_instances(None))
```

Comment:
We do. Its always CentOS server group regardless of what OS is used. I can do that here and add a comment that says "hey, this template is extremely misleading and will label any OS CentOS". It felt better to just take all the instances so this all doesn't blow up if someone puts a more accurate name than "CentosServerGroup" at that template URL.